.github/workflows/terraspace.yaml

```diff
@@ -0,0 +1,238 @@
+name: Terraspace
+
+permissions:
+  contents: read
+  id-token: write
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - terraspace/infra-prj-a/**.tf
+      - terraspace/infra-prj-a/**.tfvars
```

Comment:
With the new folder structure, you'll need to remove any references to `infra-prj-a`